t/response.t

```diff
@@ -183,3 +183,15 @@ is($r2->filename(), 'baz.txt');
 $r2->remove_header ('Content-Location');
 $req->uri('http://www.example.com/bar.txt');
 is($r2->filename(), 'bar.txt');
+
+# RFC 8187
+## Encoding: See <http://purl.org/NET/http/content-disposition-tests#encoding-2231-char>.
+$r2->header ('Content-Disposition' => "attachment; filename*=iso-8859-1''foo-%E4.html\n");
+is($r2->filename(), 'foo-ä.html', 'attwithisofn2231iso');
+$r2->header ('Content-Disposition' => "attachment; filename*=UTF-8''foo-%c3%a4-%e2%82%ac.html\n");
+is($r2->filename(), 'foo-ä-€.html', 'attwithfn2231utf8');
```

Comment:
There are non-ascii characters in this string, but `use utf8` doesn't seem to be present in this file. Consequently the file will be parsed as latin-1 which makes it more difficult to follow what exactly is going on in the test. Could you encode the file as utf8, update the characters in the test accordingly, and add `use utf8`?